schemamd/behaviors.go

```diff
@@ -7,7 +7,18 @@ import (
 // childIsRequired returns true for blocks with min items > 0 or explicitly required
 // attributes
 func childIsRequired(block *tfjson.SchemaBlockType, att *tfjson.SchemaAttribute) bool {
-	if att != nil {
+	if att.AttributeNestedType != nil {
```

Comment:
if `att` could be nil here (and based on the line lower down it seems like it could?), this would panic